x-pack/plugins/stack_connectors/common/crowdstrike/schema.ts

```diff
@@ -0,0 +1,205 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { SUB_ACTION } from './constants';
+
+// Connector schema
+export const CrowdstrikeConfigSchema = schema.object({});
+export const CrowdstrikeSecretsSchema = schema.object({
+  clientId: schema.string(),
+  clientSecret: schema.string(),
+});
+
+// TODO TC: Response Schemas - most is commented out, because we are not able to test it against a real endpoint just yet.
+export const CrowdstrikeBaseApiResponseSchema = schema.object(
+  {
+    resources: schema.arrayOf(schema.any()),
+    errors: schema.nullable(schema.arrayOf(schema.any())),
+    meta: schema.object({
+      query_time: schema.maybe(schema.number()),
+      powered_by: schema.maybe(schema.string()),
+      trace_id: schema.maybe(schema.string()),
+    }),
+  },
+  { unknowns: 'allow' }
+);
+
+export const CrowdstrikeGetAgentsResponseSchema = schema.object(
+  {
+    resources: schema.arrayOf(
+      schema.arrayOf(
+        schema.object(
+          {
+            device_id: schema.maybe(schema.string()),
+            cid: schema.maybe(schema.string()),
+            agent_load_flags: schema.maybe(schema.string()),
+            agent_local_time: schema.maybe(schema.string()),
+            agent_version: schema.maybe(schema.string()),
+            bios_manufacturer: schema.maybe(schema.string()),
+            bios_version: schema.maybe(schema.string()),
+            config_id_base: schema.maybe(schema.string()),
+            config_id_build: schema.maybe(schema.string()),
+            config_id_platform: schema.maybe(schema.string()),
+            cpu_signature: schema.maybe(schema.string()),
+            cpu_vendor: schema.maybe(schema.string()),
+            external_ip: schema.maybe(schema.string()),
+            mac_address: schema.maybe(schema.string()),
+            instance_id: schema.maybe(schema.string()),
+            service_provider: schema.maybe(schema.string()),
+            service_provider_account_id: schema.maybe(schema.string()),
+            hostname: schema.maybe(schema.string()),
+            first_seen: schema.maybe(schema.string()),
+            last_login_timestamp: schema.maybe(schema.string()),
+            last_login_user: schema.maybe(schema.string()),
+            last_login_uid: schema.maybe(schema.string()),
+            last_seen: schema.maybe(schema.string()),
+            local_ip: schema.maybe(schema.string()),
+            major_version: schema.maybe(schema.string()),
+            minor_version: schema.maybe(schema.string()),
+            os_version: schema.maybe(schema.string()),
+            platform_id: schema.maybe(schema.string()),
+            platform_name: schema.maybe(schema.string()),
+            policies: schema.maybe(
+              schema.arrayOf(
+                schema.object({
```

Comment:
Suggestion: add `{ unknowns: 'allow' }` to this `schema.object()` definition 

Same comment for others definitions below. Adding this will just ensure that if a response from the 3rd party system returns a new value that we are not aware of it will not cause the sub-actions framework to throw an error.

Reply:
did you miss this `schema.object()`? I see the `{ unknowns: 'allow' }` in other places below, but not here.